libpromises/evalfunction.c

```diff
@@ -8136,6 +8142,41 @@ void ModuleProtocol(EvalContext *ctx, char *command, const char *line, int print
         }
         break;
 
+    case '&':
+        // TODO: the variable name is limited to 256 to accommodate the
+        // context name once it's in the vartable.  Maybe this can be relaxed.
+        sscanf(line + 1, "%256[^=]=%4095[^\n]", name, content);
+
+        if (CheckID(name))
+        {
+
+            if (FileCanOpen(content, "r"))
+            {
+                int size_max = IntFromString("inf");
+                const char *requested_mode = NULL;
+
+                requested_mode =  FileType(content);
```

Comment:
Don't assign NULL, and then immediately something else. Just do both declaration and assignment in one line:

```
const char *const requested_mode = FileType(content);
```

Reply:
I will do